lib/ios/native-navigation/ReactNavigationImplementation.swift

```diff
@@ -51,15 +51,23 @@ class BlockBarButtonItem: UIBarButtonItem {
     self.target = self
   }
 
+  convenience init(barButtonSystemItem: UIBarButtonSystemItem) {
+    self.init(barButtonSystemItem: barButtonSystemItem, target: nil, action: #selector(barButtonItemPressed))
+    self.target = self
+  }
+
   convenience init(
     title: String?,
     image: UIImage?,
+    barButtonSystemItem: UIBarButtonSystemItem?,
     style: UIBarButtonItemStyle,
     enabled: Bool?,
     tintColor: UIColor?,
     titleTextAttributes: [String: Any]?
   ) {
-    if let title = title {
+    if let barButtonSystemItem = barButtonSystemItem {
+      self.init(barButtonSystemItem: barButtonSystemItem)
```

Comment:
I was just following the style of the other convenience initialisers. I personally don't see the value, but I didn't want to change it in this PR